go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/stacklok/toolhive-registry
 
-go 1.24.5
+go 1.25
```

Comment:
Does ToolHive import this at all? If so, we're gonna have some trouble. We're on 1.24 in toolhive because that's what UBI currently supports.

Reply:
ah, that's a bummer. Let's figure out something then